src/tsec.exemptions.json

```diff
@@ -49,8 +49,6 @@
 		"sql/base/browser/ui/panel/panel.ts",
 		"sql/base/browser/ui/selectBox/selectBox.ts",
 		"sql/base/browser/ui/panel/panel.component.ts",
-		"sql/workbench/services/dialog/browser/dialogModal.ts",
```

Comment:
I went through the other files we were directly setting HTML in and didn't see anything else with this issue - they're mostly just creating raw HTML elements and directly setting the contents or other seemingly-valid use cases.